src/serviceWorker/index.ts

```diff
@@ -19,31 +18,20 @@ self.skipWaiting()
 // This must be done before setting up workbox-precaching, so that it takes precedence.
 registerRoute(new DocumentRoute())
 
-// Splits entries into assets, which are loaded on-demand; and entries, which are precached.
-// Effectively, this precaches the document, and caches all other assets on-demand.
-const { assets, entries } = self.__WB_MANIFEST.reduce<{ assets: string[]; entries: PrecacheEntry[] }>(
-  ({ assets, entries }, entry) => {
-    if (typeof entry === 'string') {
-      return { entries, assets: [...assets, entry] }
-    } else if (entry.revision) {
-      return { entries: [...entries, entry], assets }
-    } else {
-      return { entries, assets: [...assets, toURL(entry)] }
-    }
-  },
-  { assets: [], entries: [] }
-)
+// Splits entries into:
+// - mediaURLs: loaded on-demand
+// - precacheEntries
+const { mediaURLs, precacheEntries } = groupEntries(self.__WB_MANIFEST)
 
-// Registers the assets' routes for on-demand caching.
 registerRoute(
   new Route(
-    ({ url }) => assets.includes('.' + url.pathname),
+    ({ url }) => mediaURLs.includes('.' + url.pathname),
     new CacheFirst({
-      cacheName: 'assets',
+      cacheName: 'media',
```

Comment:
Will this remove the old `assets` cache?

Reply:
it will because we call `skipWaiting` at the top which clears everything